apps/system/utils/utils_cpuload.c

```diff
@@ -91,21 +118,24 @@ static int cpuload_read_proc(FAR struct dirent *entryp, FAR void *arg)
 
 static void cpuload_print_tasklist(void)
 {
-	printf("==================================\n");
-	printf(" CPU USAGE\n");
-	printf("==================================\n");
-	printf(" PID");
+	printf("PID |");
+#ifdef CONFIG_SCHED_CPULOAD_SNAPSHOT
+	printf("  Snap ticks  |");
+#endif
 #ifdef CONFIG_SCHED_MULTI_CPULOAD
-	printf("%7ds %8ds %8ds", CONFIG_SCHED_CPULOAD_TIMECONSTANT_SHORT, CONFIG_SCHED_CPULOAD_TIMECONSTANT_MID, CONFIG_SCHED_CPULOAD_TIMECONSTANT_LONG);
+	printf("%5ds | %4ds | %4ds |", CONFIG_SCHED_CPULOAD_TIMECONSTANT_SHORT, CONFIG_SCHED_CPULOAD_TIMECONSTANT_MID, CONFIG_SCHED_CPULOAD_TIMECONSTANT_LONG);
 #else
-	printf("%9ds", CONFIG_SCHED_CPULOAD_TIMECONSTANT);
+	printf("%5ds |", CONFIG_SCHED_CPULOAD_TIMECONSTANT);
 #endif
-	printf("\n----------------------------------\n");
+
+	printf("\n--------------------------------------------------\n");
 
 	/* Print cpu load for each task */
 	utils_proc_pid_foreach(cpuload_read_proc);
-
-	printf("==================================\n");
+#ifdef CONFIG_SCHED_CPULOAD_SNAPSHOT
+	printf(" * Snapshot interval : %ds (%d ticks)", CPULOAD_NSNAPTICKS / CLOCKS_PER_SEC, CPULOAD_NSNAPTICKS);
```

Comment:
CPULOAD_NSNAPTICKS / CLOCKS_PER_SEC can be float. Is it okay to print with %d format?